cns/restserver/restserver.go

```diff
@@ -135,6 +137,21 @@ type containerstatus struct {
 	VfpUpdateComplete             bool // True when VFP programming is completed for the NC
 }
 
+type multiContainerStatus map[string]containerstatus
+
+func (mcs *multiContainerStatus) GetUnsuccessfulStatusErrors() string {
+	var unsuccessfulStatuses []string
+	for ncID := range *mcs {
+		ncStatus := (*mcs)[ncID].CreateNetworkContainerRequest.NCStatus
+		if ncStatus != cns.NCUpdateSuccess {
+			unsuccessfulStatus := fmt.Sprintf("Expected status for NC %s to be %s but got %s", ncID, string(cns.NCUpdateSuccess), string(ncStatus))
```

Comment:
This is good output for a test, but not for an error. Generally, it should be concise and lower-case. As in my other comment, I think something like this would be more typical:

`multiple NCs failed: uuid1 (status1), uuid2 (status2), uuid3 (status3)`

It's implied that the expectation was success, so logging that just consumes log disk space. Though that is perfectly fine (and desirable!) to mention in `go test` output.